data/json/furniture_and_terrain/terrain-floors-outdoors.json

```
@@ -148,6 +148,38 @@
     },
     "examine_action": "dirtmound"
   },
+  {
+    "type": "terrain",
+    "id": "t_paddy",
+    "name": "paddy field",
+    "description": "A damp, muddy field used to grow crops that require a lot of water.  The clay holds moisture, making it perfect for the crops.",
+    "symbol": "#",
+    "color": "brown",
+    "move_cost": 3,
+    "flags": [ "TRANSPARENT", "DIGGABLE", "VEH_TREAT_AS_BASH_BELOW", "PLANTABLE", "SHALLOW_WATER" ],
+    "bash": {
+      "sound": "thump",
+      "ter_set": "t_pit_shallow",
+      "str_min": 50,
+      "str_max": 100,
+      "str_min_supported": 100,
+      "items": [ { "item": "clay_lump", "count": [ 2, 6 ] } ]
+    },
+    "examine_action": "dirtmound",
+    "//": "Is this work even not t_dirtmound?"
+  },
+  {
+    "type": "terrain",
+    "id": "t_clay_bog",
+    "name": "ooze bog",
+    "description": "A pit filled with muddy water.  Neither used as a water source nor ground.  Keep pouring water and stirring it.",
```

Weird phrasing for second sentence, and the last sentence might confuse players as to whether to drop water on it or not.
```suggestion
    "description": "A pit filled with muddy water.  With additional work, some clay could be separated from the silt.",
```